src/components/pdf-document.js

```diff
@@ -106,8 +116,8 @@ function PdfDocument({ file, pageNumber, stretchDirection, showTextOverlay, onLo
             <Page
               key={actualPageNumber}
               pageNumber={actualPageNumber}
-              height={stretchDirection === ORIENTATION.vertical ? containerHeight : null}
-              width={stretchDirection === ORIENTATION.horizontal ? containerWidth : null}
+              height={null}
+              width={containerWidth}
```

Comment:
I went for simplifying the component by dropping this never used case. Hope that's alright for you.

Reply:
Oh, more than alright :-)